pkg/networkmanager/interfaces.js

```diff
@@ -1718,6 +1726,7 @@ PageNetworking.prototype = {
                 return;
 
             var dev = iface.Device;
+	    var zone = iface.Device.ActiveConnection.Connection.Settings.connection.zone;
```

Comment:
This will crash if any of the intermediate fields are `undefined` no?  At least `iface.Device` and `iface.Device.ActiveConnection` might be `undefined`.  `Connection.Settings.connection` is probably OK.